### PR TITLE
fix(css-prefixes): kf/chunk/1

### PR DESCRIPTION
**Pull request by firestack**
I liked the way @KaylaBrady was chunking CSS so I decided to try it, Thanks Kayla!

Each commit should be individually reviewable and a atomic update to the css classes.

Everything in this was changed over to component prefixes, I didn't see opportunities for other prefixes but I also wasn't looking super closely on the larger changesets.

---

<details>
<summary>
My change process
</summary>

I've been using `ripgrep` and `sd` (and `fish` shell) to update files using the form similar to the following snippet

```
❯ sd m-vehicle-icon c-vehicle-icon (rg m-vehicle-icon -l)
```

</details>

---

Asana Ticket: https://app.asana.com/0/1200180014510248/1203760641228313

### CODE REVIEW

File: assets/css/_ladder.scss [hunk 1]
```diff
@@ -29,29 +29,29 @@
...
       stroke: $color-secondary-light;
     }
 
-    .m-vehicle-icon__label-background {
+    .c-vehicle-icon__label-background {
       fill: $color-secondary-light;
     }
   }
 
   &.selected {
```
Comment (lemald) [inline assets/css/_ladder.scss:45]: **question:** in a few places we do have classnames like this that represent some sort of component state that don't fall into our overall naming scheme... should we do something with them or let that wait for another time?

Comment (lemald) [reply assets/css/_ladder.scss:45]: One point of view is that this is what the "M" in "BEM" is for, so it should actually be `.m-ladder__vehicle--selected`.

**Comment by lemald** [comment]
@firestack minor nitpick, but I the commit message for fee1915 mentions `c-vehicle-map-state` rather than `c-vehicle-map`, the actual class base name in question.

File: assets/src/components/routeVariantName.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const RouteVariantName = ({
   return (
     <output
       aria-label="Route Variant Name"
-      className={"m-route-variant-name " + className}
+      className={"c-route-variant-name " + className}
```
Comment (lemald) [inline assets/src/components/routeVariantName.tsx:20]: **issue (non-blocking):** I actually just noticed while QA-ing that `className` can actually be undefined, meaning that we put the literal string `"undefined"` here. If you want to fix it quickly that could be nice but if you feel like it's out of scope for this PR that's fine too.

**Review by lemald — DISMISSED** [review]
Okay, I left some non-blocking comments but otherwise this looks good and I did a bit of QA on all of the changed classes and didn't notice any issues. 👍

File: assets/css/_ladder.scss (hunk 1 above)
Comment (firestack) [reply assets/css/_ladder.scss:45]: I don't think we should handle it in this PR but I think it's reasonable to update it to match, maybe we just update them as we catch them?
Either way, I'll open a PR for this one

cc: @KaylaBrady

Comment (firestack) [reply assets/css/_ladder.scss:45]: https://github.com/mbta/skate/pull/2004
